test/OptimisticArbitrator/OptimisticArbitrator.Lifecycle.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  ExpandedERC20Ethers,
+  MockOracleAncillaryEthers,
+  OptimisticOracleV2Ethers,
+  StoreEthers,
+} from "@uma/contracts-node";
+import { OptimisticArbitrator } from "../../typechain";
+import { optimisticArbitratorFixture, seedAndApprove, umaEcosystemFixture } from "../fixtures";
+import { ethers, expect, SignerWithAddress } from "../utils";
+
+let optimisticArbitrator: OptimisticArbitrator, usdc: ExpandedERC20Ethers;
+let optimisticOracle: OptimisticOracleV2Ethers, store: StoreEthers, mockOracle: MockOracleAncillaryEthers;
+let deployer: SignerWithAddress, ancillaryData: Uint8Array, liveness: number;
+
+describe("OptimisticArbitrator: Lifecycle", function () {
+  beforeEach(async function () {
+    // Load accounts and run fixtures to set up tests.
+    [deployer] = await ethers.getSigners();
+    ({ optimisticOracle, mockOracle, store } = await umaEcosystemFixture());
+    ({ optimisticArbitrator, usdc } = await optimisticArbitratorFixture());
+
+    const amountToSeedWallets = ethers.utils.parseUnits("100000", await usdc.decimals()); // 10000 USDC
+
+    liveness = 3600; // 1 hour
+    ancillaryData = ethers.utils.toUtf8Bytes(
+      `assertion: The price of BTC was above $18000 for the duration of October 10, 2022, UTC time, considering top 5 volume weighted markets. res_data: p1: 0, p2: 1. Where p1 corresponds to False, p2 to True`
+    );
+
+    // Set the final fee in the store
+    store.setFinalFee(usdc.address, { rawValue: ethers.utils.parseUnits("1500", await usdc.decimals()) });
+
+    // Mint some fresh tokens for the deployer.
+    await seedAndApprove([deployer], usdc, amountToSeedWallets, optimisticArbitrator.address);
+  });
+
+  it("Assert happy path", async function () {
+    // get block timestamp
+    const requestTimestamp = await (await ethers.provider.getBlock("latest")).timestamp;
+    await optimisticOracle.setCurrentTime(requestTimestamp);
+
+    const balanceBefore = await usdc.balanceOf(deployer.address);
+
+    const tx = await optimisticArbitrator.makeAssertion(
+      requestTimestamp,
+      ancillaryData,
+      1,
```

Comment:
When returned, the p1-p4 values should be scaled by 1e18 as per [UMIP](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-107.md)

Reply:
Thank you! Fixed